app/models/ext_management_system.rb

```diff
@@ -45,6 +45,8 @@ def self.supported_types_and_descriptions_hash
   has_many :storages,       -> { distinct },          :through => :hosts
   has_many :ems_events,     -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "ems_id",
                                                       :inverse_of => :ext_management_system
+  has_many :carried_events, -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "source_ems_id",
```

Comment:
@chrisarcand do you have a good suggestion on how to name this?

Maybe 
```ruby
e = EventStream.new.producing_ems
e.produced_events
```

emit? generated?